src/utils/sources-tree.js

```diff
@@ -285,17 +312,18 @@ function collapseTree(node: any, depth: number = 0) {
  * @memberof utils/sources-tree
  * @static
  */
-function createTree(sources: any) {
+function createTree(sources: any, debuggeeUrl: any) {
   const uncollapsedTree = createNode("root", "", []);
   for (let source of sources.valueSeq()) {
-    addToTree(uncollapsedTree, source);
+    addToTree(uncollapsedTree, source, debuggeeUrl);
   }
   const sourceTree = collapseTree(uncollapsedTree);
 
   return { uncollapsedTree,
     sourceTree,
     parentMap: createParentMap(sourceTree),
-    focusedItem: null };
+    focusedItem: null,
+    debuggeeUrl };
```

Comment:
lets get some unit tests in here. Should be pretty similar to what we already have